index.bs

```diff
@@ -691,7 +691,7 @@ option. If {{UnderlyingSource/type}} is set to undefined (including via omission
    directly read individual [=chunks=] from the stream via its {{ReadableStreamBYOBReader/read()}}
    method, into developer-supplied buffers, allowing more precise control over allocation.
 
- <dt><code><var ignore>readable</var> = <var ignore>stream</var>.{{ReadableStream/pipeThrough(transform, options)|pipeThrough}}({ {{ReadableWritablePair/writable}}, {{ReadableWritablePair/readable}} }[, { {{StreamPipeOptions/preventClose}}, {{StreamPipeOptions/preventCancel}}, {{StreamPipeOptions/preventCancel}}, {{StreamPipeOptions/signal}} }])</code></dt>
+ <dt><code><var ignore>readable</var> = <var ignore>stream</var>.{{ReadableStream/pipeThrough(transform, options)|pipeThrough}}({ {{ReadableWritablePair/writable}}, {{ReadableWritablePair/readable}} }[, { {{StreamPipeOptions/preventClose}}, {{StreamPipeOptions/preventCancel}}, {{StreamPipeOptions/preventAbort}}, {{StreamPipeOptions/signal}} }])</code></dt>
```

Comment:
Nit-pick: looking at the definition of [StreamPipeOptions](https://streams.spec.whatwg.org/commit-snapshots/c63ae9eb7de6069f0e3cf5ea6b0ef68a2c8a3971/#dictdef-streampipeoptions) and [ReadableStreamPipeTo](https://streams.spec.whatwg.org/commit-snapshots/c63ae9eb7de6069f0e3cf5ea6b0ef68a2c8a3971/#readable-stream-pipe-to), it looks like the "intended" order should be `{ preventClose, preventAbort, preventCancel }`. Although it doesn't *really* matter that much.

Reply:
Yes, please match the order we use elsewhere.